lib/ES/Util.pm

```diff
@@ -706,7 +706,7 @@ sub timestamp {
 #===================================
 sub build_docs_js {
 #===================================
-    run '/node_modules/parcel/bin/cli.js', 'build', 'resources/web/docs_js/index.js', '/node_modules', '-d', 'resources/web', '-o', 'docs.js';
+    run '/node_modules/parcel/bin/cli.js', 'build', 'resources/web/docs_js/index.js', '/node_modules', '-d', 'resources/web', '-o', 'docs.js', '--experimental-scope-hoisting';
```

Comment:
This new flag enables tree shaking so we can cherry-pick utility functions from Ramda without adding more thank a kb or so to the final package.

Reply:
>  Tree shaking is a term commonly used in the JavaScript context for dead-code elimination. 

Thanks [webpack.js!](https://webpack.js.org/guides/tree-shaking/). Now I know a new term!